src/commonMain/kotlin/it/krzeminski/snakeyaml/engine/kmp/common/Anchor.kt

```diff
@@ -54,6 +54,6 @@ class Anchor(val value: String) {
             '*',
             '&',
         )
-        private val SPACES_PATTERN = Regex("\\s")
+        private val SPACES_PATTERN = Regex("""\s""")
```

Comment:
suggestion: I would leave such small patterns with escape characters - for me it adds more noise around in such cases. What do you think @Vampire @krzema12 ?